tests/plotting/test_plotting.py

```diff
@@ -2491,8 +2492,9 @@ def test_add_text():
 
 
 @pytest.mark.skipif(
-    not vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported(),
-    reason='Math text is not supported.',
+    tuple(map(int, matplotlib.__version__.split('.')[:2])) >= (3, 6)
+    and pyvista.vtk_version_info <= (9, 2, 2),
```

Comment:
```suggestion
    not vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported()
    or (tuple(map(int, matplotlib.__version__.split('.')[:2])) >= (3, 6)
    and pyvista.vtk_version_info <= (9, 2, 2)),
```

black may not be happy about that^